5.5/examples/replica/README.md

```diff
@@ -26,8 +26,12 @@ This template will start with one MySQL master server and one slave server.
 ## Persistent storage
 
 In order to provide the persistent storage for the MySQL master server, the administrator
-of OpenShift needs to create a PersistentVolume that you can claim. This example requires a PersistentVolume of size 512m be available.
-To learn more about how to create PersistentVolume, refer to [OpenShift documentation](https://docs.openshift.org/latest/install_config/persistent_storage/persistent_storage_nfs.html)
+of OpenShift needs to create a PersistentVolume that you can claim. This example requires two persistent volumes of size 512m be available.
+To learn more about how to create PersistentVolume, refer to the [OpenShift documentation](https://docs.openshift.org/latest/install_config/persistent_storage/persistent_storage_nfs.html).
+
+The slave use persistent storage to serve as a hot backup and to optimize
+restart performance, such that only the data written while the slave was down
+needs to be replicated and not the entire database.
```

Comment:
- One doesn't need to change the template do use emptyDir, because the template defines a PVC, not a PV.
- In real use scenarios, if you already have data on the master, replication won't work without manual intervention. See the documentation on how to enable replication when the master has initial data: https://dev.mysql.com/doc/refman/5.5/en/replication-howto-existingdata.html. It requires a dump and import. BTW this is not very different than MongoDB.